test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithDependencies.cs

```diff
@@ -147,9 +149,11 @@ public void It_publishes_projects_with_simple_dependencies_with_filter_profile()
             });
 
            var runtimeConfig = ReadJson(System.IO.Path.Combine(publishDirectory.ToString(), $"{project}.runtimeconfig.json"));
-
            runtimeConfig["runtimeOptions"]["tfm"].ToString().Should().Be("netcoreapp1.1");
-            
+
+            var depsJson = ReadJson(System.IO.Path.Combine(publishDirectory.ToString(), $"{project}.deps.json"));
+            depsJson["libraries"]["Newtonsoft.Json/9.0.1"]["runtimeStoreManifestName"].ToString().Should().Be($"{manifestFileName1} {manifestFileName2}");
+
 //TODO: Enable testing the run once dotnet host has the notion of looking up shared packages
```

Comment:
This should be true now right?  Can we get an issue logged for this TODO so we do this testing?